src/backend/distributed/commands/database.c

```diff
@@ -213,6 +244,35 @@ PreprocessAlterDatabaseRefreshCollStmt(Node *node, const char *queryString,
 
 #endif
 
+/*
+ * CreateDDLTaskList creates a task list for running a single DDL command.
+ */
+static List *
+CreateDDLTaskList(char *command, List *workerNodeList, bool outsideTransaction)
+{
+	List *commandList = list_make3(DISABLE_DDL_PROPAGATION,
+								   command,
+								   ENABLE_DDL_PROPAGATION);
+
+	Task *task = CitusMakeNode(Task);
+	task->taskType = DDL_TASK;
+	SetTaskQueryStringList(task, commandList);
+	task->cannotBeExecutedInTransction = outsideTransaction;
```

Comment:
would be nice to fix the typo in cannotBeExecutedInTransction :)

Reply:
Fixed